tests/torch/quantization/test_unified_scales.py

```diff
@@ -781,6 +780,9 @@ def test_weight_and_act_quantizer_scale_unification(self, tmp_path):
 
         unified_fq_node_inputs = [resolve_constant_node_inputs_to_values(fq_node, onnx_model.graph)
                                   for fq_node in fq_nodes_with_expected_unified_scales]
+        # delete weights from input dict
+        for item in unified_fq_node_inputs:
+            item.pop(min(item.keys()))
```

Comment:
Since quantization on export transforms weights from `Initializer `to `Constant `type of input to onnx node they are incorrectly (w.r.t. test logic which compares quantizer scales) being found by `resolve_constant_node_inputs_to_values` along with quantizer scales. So I decided to manually remove weights from constant node inputs dict to keep test running. Weight inputs to onnx nodes have the lowest ID of all other inputs, `min(item.keys())` gives ID of weighs input and allows to remove weights by ID, so they don't ruin test logic.